README.md

```diff
@@ -10,6 +10,7 @@ An easy way to send emails with stack trace whenever an exception occurs on the
 
 #### For Laravel <= 5.2, please use the [v1 branch](https://github.com/squareboat/sneaker/tree/v1)!
 #### For Laravel 5.2 < version <= 6.x, please use the [v5 branch](https://github.com/squareboat/sneaker/tree/v5)!
+#### For Laravel 6.x < version <= 8.x, please use the [v7 branch](https://github.com/squareboat/sneaker/tree/v7)!
```

Comment:
This line is a bit confusing because of the .x at the start. 
I recommend changing it into "For Laravel 7.0 <= version <= 8.x" 😊
And while you're at it, I recommend also changing that second line into "For Laravel 5.3 <= version ..." then...

Less brain cells wasted for developers trying to figure out which branch they need 😊